docs/content/en/_index.html

```diff
@@ -38,7 +38,7 @@
             <li>Automated deployment analysis based on metrics, logs, emitted requests</li>
             <li>Automatically roll back to the previous state as soon as analysis or a pipeline stage fails</li>
             <li>Automatically detect configuration drift to notify and render the changes</li>
-            <li>Automatically watch and detect the new container images to deploy</li>
+            <li>Automatically update configuration files when it finds out a new event</li>
```

Comment:
I agree that this is a correct sentence to describe what EventWatcher does.
But I think because this is the landing page, so maybe we need more catchy words. 😄 

"Automatically trigger deployment for your specified events include container image pushing, helm chart publishing..."

I hope that you guys have better sentences.